internal/broker/bind.go

```diff
@@ -3,36 +3,328 @@ package broker
 import (
 	"context"
 	"fmt"
+	"net/http"
+	"sync"
 
+	jsonhash "github.com/komkom/go-jsonhash"
 	"github.com/pkg/errors"
 	osb "github.com/pmorie/go-open-service-broker-client/v2"
+	"github.com/sirupsen/logrus"
+	"k8s.io/helm/pkg/proto/hapi/chart"
 
 	"github.com/kyma-project/helm-broker/internal"
 )
 
 type bindService struct {
-	instanceBindDataGetter instanceBindDataGetter
+	addonIDGetter                 addonIDGetter
+	chartGetter                   chartGetter
+	instanceGetter                instanceGetter
+	bindTemplateRenderer          bindTemplateRenderer
+	bindTemplateResolver          bindTemplateResolver
+	instanceBindDataInserter      instanceBindDataInserter
+	instanceBindDataGetter        instanceBindDataGetter
+	instanceBindDataRemover       instanceBindDataRemover
+	bindStateGetter               bindStateBindingGetter
+	bindOperationGetter           bindOperationGetter
+	bindOperationCollectionGetter bindOperationCollectionGetter
+	bindOperationInserter         bindOperationInserter
+	bindOperationUpdater          bindOperationUpdater
```

Comment:
I know that bindOperationRemover is not used in this file but maybe it's better to use `bindOperationStorage` here?